src/applications/personalization/account/components/ConnectedAccountsSection.jsx

```diff
@@ -10,25 +12,60 @@ function recordAnalyticEvent() {
   });
 }
 
-export default function ConnectedAccountsSection() {
-  return (
-    <div>
-      <h3>Connected accounts</h3>
+function ConnectedAccountsSection({
+  accounts,
+  loadConnectedAccounts,
+  loading,
+}) {
+  // TODO: for some reason this hook is called twice on load. maybe
+  // AccountMain's render is to blame because it renders an instance of this
+  // component and then re-renders, replacing it?
```

Comment:
It could be something like that, but also it's tough to rely on function identity as a way to keep something from rendering. It's easy to write code where the function identity changes every render call. It could be that react-redux is doing that for the `loadConnectedAccounts`.

I think you could try passing `[]` as the second argument. That's what the docs say to do to make it run just on mount, I think.

Reply:
Oops, missed your other comment about trying that already.

Reply:
I tried that with the same results ¯\_(ツ)_/¯